Composer/packages/server/src/utility/getVersion.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import path from 'path';
+
+import fs from 'fs-extra';
+
+const isProduction = process.env.NODE_ENV === 'production';
+
+export function getVersion(): string {
+  try {
+    const version = fs.readJSONSync(path.join(__dirname, '../../../electron-server/package.json')).version;
```

Comment:
does this file exist in packed electron?

Reply:
No, but the env variable gets set in the electron start up sequence. So in that case, this will just error and return `unknown`.

Right now, there is not a lazy evaluation for getting default values using the `setEnvDefault` function.